docs/components.md

```diff
@@ -175,13 +175,13 @@ class CreatePost extends Component
     public function render()
     {
         return view('livewire.create-post')->with([
-	        'author' => auth()->user()->name,
+	        'author' => Auth::user()->name,
```

Comment:
```suggestion
	        'author' => auth()->user()->name,
```
this was fine.
With`Auth::user()` you also need to import the class

Reply:
`Auth::user()` is used everywhere else, so I wanted to be consistent with that.